pkg/repository/backend_postgres.go

```diff
@@ -773,8 +773,8 @@ func (r *PostgresBackendRepository) GetOrCreateStub(ctx context.Context, name, s
 	return stub, nil
 }
 
-func generateGroupName(name, typ, workspaceID string) string {
-	nameParts := strings.Split(name, ":")
+func generateGroupName(stubName, stubType, workspaceId string) string {
+	nameParts := strings.Split(stubName, ":")
```

Comment:
This addresses a comment from the first subdomain PR. No actual logic changes.

Reply:
👍